.github/actions/collect-info/action.yml

```diff
@@ -1,20 +1,14 @@
 name: 'Collect and store debug info'
-description: 'Collect debug info using EVE script executed via ssh and store downloaded tarball under the specified file name'
+description: 'Collect debug info using EVE script executed via ssh or console and store downloaded tarball under the specified file name'
 
 runs:
   using: 'composite'
   steps:
     - name: Collect info
       run: |
-        # Give EVE 5 minutes at most to enable ssh access (if tests failed early).
-        for i in $(seq 60); do ./eden eve ssh && break || sleep 5; done
-        ./eden sdn fwd eth0 22 --\
-          ssh -o StrictHostKeyChecking=no -p FWD_PORT -i ./dist/default-certs/id_rsa root@FWD_IP collect-info.sh &&\
-        ./eden sdn fwd eth0 22 --\
-          scp -o StrictHostKeyChecking=no -P FWD_PORT -i ./dist/default-certs/id_rsa root@FWD_IP:/persist/eve-info-* . &&\
-        # upload-artifact complains about colon in the file name
-        # make sure to update upload step if changing name
-        mv eve-info-* eve-info.tar.gz ||\
-        echo "failed to collect info"
+        # Do not pollute console logs which are collected by publish-logs action.
+        cp dist/default-eve.log dist/default-eve.log.backup || true
```

Comment:
`cp dist/default-eve.log dist/default-eve.log.backup || true \\`
and more below

Reply:
Is that really necessary? We do not use backslashes in other places, e.g.: https://github.com/lf-edge/eden/blob/master/.github/workflows/eden.yml#L75-L80

Reply:
I guess we can skip it here, do you want to print something in case cp fails?